### DISCUSSION TITLE
How does FontTools decode byte-strings in name table?

### DISCUSSION THREAD

**Discussion by http403** [fonttools/fonttools#2638] [Q&A] [answered]
I'm trying to get the font name in the `name` table, but some aren't encoded with any UTF standard despite the `unicode` flag is set to `true` and `platformID` is `1`. However, saveXML have decoded them correctly. I wonder how. I tried dive into the code but I can't get a sense of it.

**Comment by behdad** [comment, marked as answer]
https://github.com/fonttools/fonttools/blob/feb4e75469910d239fb72d80bbc979a121a6f87c/Lib/fontTools/ttLib/tables/_n_a_m_e.py#L456-L505

**Reply by http403** [reply]
Thank you
